local.js

```diff
@@ -0,0 +1,23 @@
+const email = document.getElementById('email');
+const userName = document.getElementById('userName');
+const textMessage = document.getElementById('textMessage');
+const form = document.getElementById('form');
+form.addEventListener('input', () => {
+  const localStorageObject = {
+    name: userName.value,
+    email: email.value,
+    message: textMessage.value,
+  };
+  localStorage.setItem('templateFormData', JSON.stringify(localStorageObject));
+});
+const getData = JSON.parse(localStorage.getItem('templateFormData'));
+function preloaded() {
+  if (getData) {
+    userName.value = getData.name;
+    email.value = getData.email;
+    textMessage.value = getData.message;
+    return getData;
+  }
+  return '';
+}
+preloaded();
```

Comment:
- Kindly fix this error displayed in console:
<img width="1440" alt="Screen Shot 2021-11-11 at 17 40 47" src="https://user-images.githubusercontent.com/22258274/141384478-baf67eac-835a-4131-884f-826b3e7fb019.png">